docs/apis/python_api/conf.py

```diff
@@ -32,7 +32,7 @@
 ]
 
 # templates_path = ['_templates']
-exclude_patterns = []
+exclude_patterns = ["autoapi/*/_rust/*"]
```

Comment:
pyiに対してそのままちゃんと動くようになったはいいものの、なんか`_rust.{blocking,asyncio}`がorphanだぞと赤文字でwarningを出し始めたため除外設定。

Reply:
この意図あとあと忘れてしまいそうなので、雑にコメント残しといてもいいかも？
```suggestion
exclude_patterns = ["autoapi/*/_rust/*"]  # _rust.{blocking,asyncio}がwarningになってしまうため除外
```
みたいな！

Reply:
こうしようと思います。

```suggestion
exclude_patterns = ["autoapi/*/_rust/*"] # パブリックAPIを意図した部分ではなく、またorphan扱いとなって警告が出るため
```